colormath/chromatic_adaptation.py

```diff
@@ -25,13 +25,13 @@ def _get_adaptation_matrix(wp_src, wp_dst, observer, adaptation):
 
     # In case the white-points are still input as strings
     # Get white-points for illuminant
-    if type(wp_src) == str:
+    if isinstance(wp_src, str):
```

Comment:
Worth considering `basestring` instead?

Reply:
`basestring` does not exist in Python 3.x so we would need to either account for it or use `six` which is cleaner and more portable.